src/courseware/course/sidebar/sidebars/outline/OutlineSidebar.jsx

```diff
@@ -4,11 +4,27 @@ import React, { useContext } from 'react';
 import SidebarBase from 'courseware/course/sidebar/common/SidebarBase';
 import SidebarContext from 'courseware/course/sidebar/SidebarContext';
 import { ID } from 'courseware/course/sidebar/sidebars/outline/OutlineTrigger';
+import { postEventToIframe } from 'utils';
 
 import messages from './messages';
 
 ensureConfig(['SIDEBAR_MFE_BASE_URL']);
 
+/**
+ * Helper function to make a postEvent call to Outline Navigation Sidebar MFE.
+ */
+export function postEventOutlineToSidebar() {
+  const iframe = document.getElementById('OutlineSidebar');
+  const message = 'outline_event';
+  const messageTargets = [getConfig().SIDEBAR_MFE_BASE_URL];
+
+  postEventToIframe(
+    iframe,
+    message,
+    messageTargets,
+  );
+}
+
```

Comment:
What do you think about moving this function to the utils file? So this file only exports the component